debian/focal/foreman-installer/changelog

```diff
@@ -0,0 +1,77 @@
+foreman-installer (2.5.0-1) stable; urgency=low
+
+  * Bump changelog to 2.5.0 to match VERSION
+
+ -- Evgeni Golov <evgeni@debian.org>  Tue, 02 Feb 2021 11:18:07 +0100
+
+foreman-installer (2.4.0-1) stable; urgency=low
+
+  * Bump changelog to 2.4.0 to match VERSION
+
+ -- Patrick Creech <pcreech@redhat.com>  Mon, 02 Nov 2020 15:19:19 -0500
+
+foreman-installer (2.3.0-1) stable; urgency=low
+
+  * Bump changelog to 2.3.0 to match VERSION
+
+ -- Eric D. Helms <ericdhelms@gmail.com>  Tue, 11 Aug 2020 08:47:01 -0400
+
+foreman-installer (2.2.0-1) stable; urgency=low
+
+  * Bump changelog to 2.2.0 to match VERSION
+
+ -- Eric D. Helms <ericdhelms@gmail.com>  Wed, 13 May 2020 14:29:17 -0400
+
+foreman-installer (2.1.0-1) stable; urgency=low
+
+  * Bump changelog to 2.1.0 to match VERSION
+
+ -- Tomer Brisker <tbrisker@gmail.com>  Thu, 13 Feb 2020 19:00:20 +0200
+
+foreman-installer (2.0.0-1) stable; urgency=low
+
+  * Bump changelog to 2.0.0 to match VERSION
+
+ -- Tomer Brisker <tbrisker@gmail.com>  Mon, 06 Jan 2020 19:56:20 +0200
+
+foreman-installer (1.25.0-1) stable; urgency=low
+
+  * Bump changelog to 1.25.0 to match VERSION
+
+ -- Michael Moll <mmoll@mmoll.at>  Tue, 29 Oct 2019 22:43:25 +0100
+
+foreman-installer (1.24.0-1) stable; urgency=low
+
+  * Bump changelog to 1.24.0 to match VERSION
+
+ -- Evgeni Golov <evgeni@debian.org>  Tue, 30 Jul 2019 12:34:08 +0200
+
+foreman-installer (1.23.0-1) stable; urgency=medium
+
+  * Bump changelog to 1.23.0 to match VERSION
+
+ -- Evgeni Golov <evgeni@debian.org>  Tue, 23 Apr 2019 15:05:04 +0200
+
+foreman-installer (1.22.0-1) stable; urgency=low
+
+  * Bump changelog to 1.22.0 to match VERSION
+
+ -- Michael Moll <mmoll@mmoll.at>  Thu, 17 Jan 2019 01:16:09 +0100
+
+foreman-installer (1.21.0-1) stable; urgency=low
+
+  * Bump changelog to 1.21.0 to match VERSION
+
+ -- Michael Moll <mmoll@mmoll.at>  Sun, 21 Oct 2018 02:08:17 +0200
+
+foreman-installer (1.20.0-1) stable; urgency=low
+
+  * Bump changelog to 1.20.0 to match VERSION
+
+ -- Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>  Tue, 17 Jul 2018 18:26:17 +0200
+
+foreman-installer (1.19.0-1) stable; urgency=low
+
+  * Initial release for Ubuntu/bionic
```

Comment:
Looking at this, we didn't copy it in the past. I think you mentioned that you'd prefer to end up with the same directories where possible. Is this still your intention and why you copied the changelog too?

Reply:
Yeah, one day I want to get rid of the (now three) two folders.